snuba/web/rpc/v1/endpoint_trace_item_table.py

```diff
@@ -80,6 +87,48 @@ def _transform_request(request: TraceItemTableRequest) -> TraceItemTableRequest:
     return SparseAggregateAttributeTransformer(request).transform()
 
 
+def convert_to_conditional_aggregation(in_msg: TraceItemTableRequest) -> None:
```

Comment:
How different would this function be for the TimeSeries endpoint?

Reply:
1. leave a comment explaining why this is being done. It's not immediately clear unless you are the user
2. Test this function independently. It has clearly defined inputs and outputs